### PR TITLE
BYO RHEL docs need to be updated to include RHEL 7.9 support 4.5

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1918082

Updates to match supported versions shown here: https://access.redhat.com/articles/4763741

### CODE REVIEW

**Comment by mburke5678** [comment]
@gpei Can you please take a look at this PR

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -132,7 +132,7 @@ ifndef::ibm-power[|4]
 
 |Compute
 ifdef::ibm-z,ibm-power[|{op-system}]
-ifndef::ibm-z,ibm-power[|{op-system} or RHEL 7.6]
+ifndef::ibm-z,ibm-power[|{op-system} or RHEL 7.7-7-8]
```
Comment (gpei) [inline modules/installation-requirements-user-infra.adoc:135]: `7-8` typo here?

**Review by gpei — COMMENTED** [review]
Yes, RHEL 7.7-7.8 were the OS QE tested in OCP-4.5 release cycle. After RHEL-7.9 GA, RHEL-7.9 was also tested with OCP-4.5

**Comment by sdodson** [comment]
I think @gpei was saying that 7.9 should be included in the version ranges supported. Not sure.

**Comment by mburke5678** [comment]
> I think @gpei was saying that 7.9 should be included in the version ranges supported. Not sure.

@sferich888 Any thoughts on Gaoyun's comment that they tested RHEL 7.9 with 4.5? I should think we want the docs and the KCS article to match. The KCS has 7.7 and 7.8 as supported for 4.5.  
https://access.redhat.com/articles/4763741

**Pull request merged**